src/presentations/indexable-attachment-presentation.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Presentation object for indexables.
+ *
+ * @package Yoast\YoastSEO\Presentations
+ */
+
+namespace Yoast\WP\Free\Presentations;
+
+/**
+ * Class Indexable_Attachment_Presentation
+ */
+class Indexable_Attachment_Presentation extends Indexable_Post_Type_Presentation {
+
+	/**
+	 * Generates the open graph images.
+	 *
+	 * @return array The open graph images.
+	 */
+	public function generate_og_images() {
+		if ( $this->model->og_image_id === null && $this->model->og_image ) {
+			return [ $this->model->og_image ];
+		}
+
+		if ( $this->model->og_image_id ) {
```

Comment:
Consideration: if you move this if statement to above the above if statement, you can remove the `$this->model->og_image_id === null` in that if statement. This means we have to do one check less.